priv/repo/migrations/20220121222855_create_wines.exs

```diff
@@ -0,0 +1,14 @@
+defmodule WineOInventoryApi.Repo.Migrations.CreateWines do
+  use Ecto.Migration
+
+  def change do
+    create table(:wines) do
+      add :name, :string, null: false
```

Comment:
Should this be unique? Or is it ok to have different wines with the same name?

Reply:
Debatable! Typically wines can have the same name because they are named after the vineyard/company but are made from different wine grapes. For example, you might have a Josh Cellar Pinot Noir and a Jose Cellar Merlot. I was planning on adding a category section later to differentiate the different wines.